docs/scos/dev/glue-api-guides/202304.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.md

```diff
@@ -0,0 +1,110 @@
+        "minLength": 2
+      }
+    }
+  ]
+}
+```
+
+And now, let's delve into the meaning of each field within the snippet. Here's a breakdown of the purpose of each field:
+
+| FIELD | SPECIFICATION |
+| --- | --- |
+| identifier | The name of the column in the table that serves as an identifier. It can be any chosen column from the table, typically used as a unique identifier for each record. |
+| fields | An array containing the descriptions of the columns from the table. It allows customization of which columns are included for interaction. By specifying the desired columns in the "fields" array, the API will only expose and operate on those specific columns. Any columns not included in the array will be inaccessible through API requests. |
+| fieldName | The actual name of the column in the database table. |
+| fieldVisibleName | The name used for interacting with the field through API requests. It provides a more user-friendly and descriptive name compared to the physical column name. |
+| type | The data type of the field. It specifies the expected data format for the field, enabling proper validation and handling of values during API interactions. |
+| isEditable | A flag indicating whether the field can be modified. When set to "true," the field is editable, allowing updates to its value. |
+| isCreatable | A flag indicating whether the field can be set. If set to "true," the field can be included in requests to provide an initial value during record creation. |
+| validation | Contains validation configurations. Proper validation ensures that the provided data meets the specified criteria. |
+| required | A validation attribute that determines whether the field is required or optional. When set to "true," the field must be provided in API requests. |
+| maxLength/minLength | An optional validation attribute that specifies the minimum/maximum length allowed for the field defined with a string type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. |
+| max/min | An optional validation attribute that specifies the minimum/maximum value allowed for the field defined with an integer type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. |
+
+{% info_block infoBox %}
+
+It is recommended to set `isEditable` and `isCreatable` to `false` for fields that serve as identifiers or keys, ensuring their immutability and preserving the integrity of the data model.
+
+{% endinfo_block %}
+
+{% info_block infoBox %}
+
+Configuring the definition for the field responsible for the identifier keep in mind that it JSON accepts numbers within 
+the range of a 64-bit signed integer (-9,223,372,036,854,775,808 to 9,223,372,036,854,775,807). However, if there is a need 
+to use identifiers outside this range or if the person providing the configuration anticipates larger values, the field can be set as a 
+string type instead.
```

Comment:
1. It is applicable not only to identifiers but to any bigint columns.
2. It is not JSON related limitation, see https://www.w3schools.com/php/php_numbers.asp#:~:text=PHP%20Integers&text=An%20integer%20data%20type%20is,9223372036854775807%20in%2064%20bit%20systems. for 32b systems.